test_conformance/mem_host_flags/checker.h

```diff
@@ -130,6 +130,11 @@ cBuffer_checker<T>::cBuffer_checker(cl_device_id deviceID, cl_context context,
 template < class T >
 cBuffer_checker<T>::~cBuffer_checker()
 {
+    clReleaseMemObject(m_buffer);
+    if (m_buffer_parent)
+    {
+        clReleaseMemObject(m_buffer_parent);
+    }
```

Comment:
Hmm, just spotted this: both of these are `clMemWrapper`. You shouldn't have to call `clReleaseMemObject` on them. This looks suspicirous.

Reply:
yeap, I though I reverted this bit. Let me retest to make sure, everything still works